src/System.Windows.Forms/tests/AccessibilityTests/DataBindingExample.cs

```diff
@@ -0,0 +1,144 @@
+using System.Collections.Generic;
+using System.Data;
+using System.Windows.Forms;
+
+namespace AccessibilityTests
+{
+    public partial class DataBindingExample : Form
+    {
+        private readonly List<Student> _studentA = new List<Student>();
+        private readonly List<Student> _studentB = new List<Student>();
+
+        public DataBindingExample()
+        {
+            InitializeComponent();
+        }
+
+        private void Binding_Data_Load(object sender, EventArgs e)
+        {
+            for (int i = 0; i < 6; i++)
+            {
+                _studentA.Add(new Student(i, "Name 1" + i, "Male"));
+                _studentB.Add(new Student(i * 2, "Name 11" + i * 2, "Female"));
+            }
+
+            // Binding Data For ListBox & ComboBox & CheckedListBox controls by using DataSource property
+            listBox1.DataSource = _studentA;
+            comboBox1.DataSource = _studentB;
+
+            listBox1.DisplayMember = "StudentName";
+            comboBox1.DisplayMember = "StudentName";
+
+            // Binding Data For DataGridView control by using DadSource property
+            dataGridView1.DataSource = new List<Student>{
+                 new Student(1, "StudentA", "Female", 12121, "1001","Basketball",false, 10, 11),
+                 new Student(2, "StudentB", "Male", 12122, "1002","Basketball",true, 10, 11),
+                 new Student(3, "StudentC", "Female", 12123, "1003","Football",false, 10, 11),
+                 new Student(4, "StudentD", "Male", 12124,"1004","Football",true, 10, 11),};
+
+            // Binding Data For TextBox/Label/DomianUpDown/NumericUpDown/LinkLabel/CheckBox/RadioButton/RichTextBox/MaskedTextBox/Button controls by using DadaBindings property
+            Student stu = new Student(1, "StudentNumber", "Female", 12121, "HomeNumber", "Habits\nBasketball\nFootball", true, 10, 11);
+            this.textBox1.DataBindings.Add("Text", stu, "StudentNo");
+            this.domainUpDown1.DataBindings.Add("Text", stu, "Lucky_Number");
+            this.numericUpDown1.DataBindings.Add("Text", stu, "Student_Count");
+            this.label1.DataBindings.Add("Text", stu, "StudentName");
+            this.button1.DataBindings.Add("Text", stu, "StudentSex");
+            this.maskedTextBox1.DataBindings.Add("Text", stu, "StudentPhoneNum");
+            this.linkLabel1.DataBindings.Add("Text", stu, "HomeNumber");
+            this.richTextBox1.DataBindings.Add("Text", stu, "Student_habit");
+            this.checkBox1.DataBindings.Add("Checked", stu, "Is_Student");
+            this.radioButton1.DataBindings.Add("Checked", stu, "Is_Student");
+
+            // Binding Data For TreeView control by using DataSet
+            BindTree();
+
+            // Binding Data For ListView control by using DataSet
+            BindListView();
+        }
+
+        // Binding Data For TreeView control by using DataSet
+        public void BindTree()
+        {
+            DataSet ds = CreateDataSet();
```

Comment:
Instead of calling `Dispose` explicitly it is better to employ ["using"-pattern](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement).
```suggestion
            using DataSet ds = CreateDataSet();
```

and remove `ds.Dispose()` below